react_paragraphs.install

```diff
@@ -0,0 +1,368 @@
+        $key_value[$field_name]['type'] = 'entity_reference_revisions';
+      }
+      /** @var \Drupal\field\FieldStorageConfigInterface $field_storage */
+      $field_storage = FieldStorageConfig::load("$entity_type.$field_name");
+
+      $tables[] = $storage->getTableMapping()->getFieldTableName($field_name);
+      $tables[] = $storage->getTableMapping()
+        ->getDedicatedRevisionTableName($field_storage);
+      $table_data = [];
+
+      foreach ($tables as $table) {
+        $database->schema()->dropField($table, "{$field_name}_settings");
+
+        $query = $database->select($table, 't')->fields('t')->execute();
+        while ($row = $query->fetchAssoc()) {
+          $table_data[$table][] = $row;
+        }
+        $database->truncate($table)->execute();
+      }
+
+      $config_factory->getEditable("field.storage.$entity_type.$field_name")
+        ->set('type', 'entity_reference_revisions')
+        ->set('settings.target_type', 'paragraph_row')
+        ->set('module', 'entity_reference_revisions')
+        ->set('cardinality', -1)
+        ->save();
+      FieldStorageConfig::load("$entity_type.$field_name")->save();
+
+      foreach ($bundles_info['bundles'] as $bundle) {
+        $paragraph_row_bundle = "{$entity_type}_{$bundle}_row";
+
+        $config_factory->getEditable("field.field.$entity_type.$bundle.$field_name")
+          ->set('field_type', 'entity_reference_revisions')
+          ->set('settings.handler', 'default:paragraph_row')
+          ->set('settings.handler_settings.target_bundles', [$paragraph_row_bundle => $paragraph_row_bundle])
+          ->save();
+
+        $field = FieldConfig::loadByName($entity_type, $bundle, $field_name);
+        $field->set('field_type', 'entity_reference_revisions');
+        $field->save();
+
+        /** @var \Drupal\Core\Entity\Display\EntityFormDisplayInterface $form_display */
+        $form_display = EntityFormDisplay::load("$entity_type.$bundle.default");
+        $form_display->removeComponent($field_name);
+        $form_display->setComponent($field_name, [
+          'type' => 'react_paragraphs',
+          'settings' => ['resizable' => FALSE],
+        ]);
+        $form_display->save();
+      }
+
+      foreach ($table_data as $table => $rows) {
+        foreach ($rows as $row) {
+          $database->insert($table)->fields($row)->execute();
+        }
+      }
+    }
+    $key_value_factory->get('entity.definitions.bundle_field_map')
+      ->set($entity_type, $key_value);
+  }
+  \Drupal::cache()->delete('entity_field_map');
+}
+
+/**
+ * Create new rest resource.
+ */
+function react_paragraphs_update_8210() {
+  RestResourceConfig::create([
+    'id' => 'entity.paragraph_row',
+    'plugin_id' => 'entity:paragraph_row',
+    'granularity' => RestResourceConfigInterface::RESOURCE_GRANULARITY,
+    'configuration' => [
+      'methods' => ['GET'],
+      'formats' => ['json'],
+      'authentication' => ['cookie'],
+    ],
+  ])->save();
```

Comment:
Should this be optional?

Reply:
No it's needed for the functionality. the config is in `config/optional` since other modules/sites might have it already enabled.